pkg/cmd/pulumi/util.go

```diff
@@ -919,15 +920,21 @@ func fprintJSON(w io.Writer, v interface{}) error {
 
 // updateFlagsToOptions ensures that the given update flags represent a valid combination.  If so, an UpdateOptions
 // is returned with a nil-error; otherwise, the non-nil error contains information about why the combination is invalid.
-func updateFlagsToOptions(interactive, skipPreview, yes bool) (backend.UpdateOptions, error) {
+func updateFlagsToOptions(lockTimeout string, interactive, skipPreview, yes bool) (backend.UpdateOptions, error) {
 	if !interactive && !yes {
 		return backend.UpdateOptions{},
 			errors.New("--yes must be passed in non-interactive mode")
 	}
 
+	lockTimeoutDuration, err := time.ParseDuration(lockTimeout)
+	if err != nil && lockTimeout != "" {
+		return backend.UpdateOptions{}, err
```

Comment:
```suggestion
		return backend.UpdateOptions{}, fmt.Errorf("--lock-timeout invalid: %w")
```

Reply:
I'm assuming that you meant to pass `err` to `fmt.Errorf` so that's what I've done